src/Controller/OverviewController.php

```diff
@@ -90,7 +94,7 @@ public function show_all(): Response
     $announcement = $this->getDoctrine()
       // inits the database and table Announcements;
       ->getRepository(Announcement::class)
-      ->find_today();
+      ->find_today_approved();
```

Comment:
You should be able to go back to `find_today()` since that defaults to only finding approved announcements.